microsoft-edge/webview2/release-notes.md

```diff
@@ -32,9 +32,58 @@ For more information, see [Matching the Runtime version with the SDK version](co
 
 To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime is 86.0.616.0.  The minimum version to load WebView2 only changes when a breaking change occurs in the web platform.
 
-To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
+To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
+
+<!-- ====================================================================== -->
 
+## 1.0.1210.39
+  
+Release Date: May 6, 2022  
+  
+[NuGet package for WebView2 SDK 1.0.1210.39](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1210.39)  
+  
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime 101.1210.39 or higher.  
+
+### General
 
+#### Promotions
+
+The following items are now stable:
+
+*   Support for [multiple user profiles](#) in WebView2.
+
+*	[Theming API](#) which provides a way to customize the WebView2 color theme as `light`, `dark`, or `system`.
```

Comment:
Recommend Writer or template Dev normalize/reduce the number of leading spaces between asterisk and body of each list item.  Linter-like pass.  Markdown likes consistent number of leading spaces within a given .md file.  

Recommend full-text search " \n" with Regex On, in VS Code, to remove trailing spaces from lines.  per periodic cleanup of markdown tagging.

Reply:
fixed